packages/backend/src/config/getDeliveryServiceProperties.test.ts

```diff
@@ -89,23 +74,28 @@ describe('ReadDeliveryServiceProperties', () => {
                     },
                 },
             ],
-            smtpHost: 'smtp.host',
-            smtpPort: 587,
-            smtpEmail: 'dm3@gmail.com',
-            smtpUsername: 'dm3@gmail.com',
-            smtpPassword: 'dm312345',
         });
     });
     it('Adds default properties if config.yml is not fully specified', () => {
```

Comment:
This test is supposed to test, if the config reader adds proper default values when notificationsChannels are not defined in the config.yml. 
The expected outcome would be that the notification channel would be an empty array.

Plase then add another test to ensure that an email related channel can be added using the config.yml

Reply:
ok